api/models.py

```diff
@@ -484,6 +484,7 @@ class ProductStatus(models.TextChoices):
     )
     ts = SearchVectorField(null=True)
     bbox = settings.DEFAULT_EXTENT
+    max_order_area = models.IntegerField(_("max_order_area"), default=settings.MAX_ORDER_AREA)
```

Comment:
Can it happen that settings.MAX.ORDER_AREA does not exist?
I would like to avoid an error in that case, and simply use 0 as value.

Reply:
I think that this is already the case @jwkaltz  
https://github.com/camptocamp/geoshop-back/blob/GSGGR-403/default_settings.py#L384

Reply:
If there is no env variable, then settings.MAX_ORDER_AREA is set to 0, so it's always there